### ISSUE TITLE
EPUB Accessibility Conformance: WCAG-a/AA is too specific to EPUB accessibility

### ISSUE THREAD

**Issue by avneeshsingh** [w3c/publ-a11y#4]
The following recommendation for showing accessibility conformance metadata is too specific to EPUB Accessibility:
EPUB Accessibility Conformance: WCAG-AA

This does not make this guidance document future proof. In future there can be a new publishing standard with a new name. There is also a possibility to develop the successor of EPUB Accessibility specifications, without such a close binding with EPUB 3 standards.
Therefore it is recommended to have a neutral label on left side and EPUB specific conformance information on right side for example:
Accessibility Conformance: EPUB Accessibility (WCAG AA)

This is just an example, we can improve it.

**Comment by avneeshsingh** [comment]
Decision made in February 14, 2020 meeting:

Replace the heading "EPUB Accessibility Conformance" by "Accessibility Conformance".

Lets finalize its values on this thread. Some values discussed were:

Accessibility Conformance: WCAG A (if the publication conforms only to WCAG and not to EPUB Accessiblity)

Accessibility Conformance: EPUB Accessibility - A (If the publication   conforms to EPUB accessibility specs)

**Comment by mattgarrish** [comment]
> Accessibility Conformance: EPUB Accessibility - A

The danger with this, as we discussed while developing the specification, is that it suggests we have our own levels distinct from WCAG. I'd prefer to list each conformance:

> Accessibility Conformance: EPUB Accessibility, WCAG AA

Or even list each separately:

> Accessibility Conformance: EPUB Accessibility
> Accessibility Conformance: WCAG AA

**Comment by GeorgeKerscher** [comment]
My concern is that people would list only WCAG A or AA and ignore the EPUB specific requirements. Is it possible to include a couple listing the accessibility of WCAG and EPUB? Perhaps EPUB could be a true or false. Of course the problem is if that is not included. I would think that the EPUB part must be required with either a true or false value.

**Comment by clapierre** [comment]
So there are 2 issues here the label/Heading and its value.

1. Original Label/Heading was "EPUB Accessibility Conformance" becomes "Accessibility Conformance"

2. the Value was "WCAG-A" "WCAG-AA" becomes ?

@mattgarrish  suggested "EPUB Accessibility, WCAG AA" or two separate statements but @GeorgeKerscher does not like the idea of two separate statements since folks may only put in WCAG-AA and forget about EPUB Accessibility.

I am fine with the combination of both myself.
So it would look like

Accessibility Conformance: EPUB Accessibility, WCAG A
or
Accessibility Conformance: EPUB Accessibility, WCAG AA

And for something like an optimized publication like an Audio book maybe:
Accessibility Conformance: EPUB Accessibility, DAISY Audio 
but maybe this wouldn't have this EPUB Accessibility portion since that may not make sense for optimized publications like an Audio book and it may just be
Accessibility Conformance:DAISY Audio

**Comment by gregoriopellegrino** [comment]
During the call we decided to use last @clapierre solution:

> Accessibility Conformance: EPUB Accessibility, WCAG A
> or
> Accessibility Conformance: EPUB Accessibility, WCAG AA

**Comment by gregoriopellegrino** [comment]
I was working to fix this issue, but I realized that the definition of metadata in the principles document needs to be rethought, because it refers only to the Schema.org case and doesn't fit with what we mean.

https://w3c.github.io/publ-a11y/UX-Guide-Metadata/principles/#accessibility-conformance

My proposal is:

**Value**: list of links, based on metadata

**Definition**
This metadata reports the conformity of the publication to one or more accessibility standards. The different standards can be listed comma separated.
In the case of EPUBs, if they conforms to EPUB Accessibility Guidelines, it means that they also conforms to WCAG (on which they are based). In this case the double value must be reported, for example: _Accessibility Conformance: EPUB Accessibility, WCAG AA_

**From the note remove**: In this case, the conformsTo term must point to a URL that identifies the specification used to create the optimized publication.

**Comment by clapierre** [comment]
We added that incase this publication is for example an audiobook ie "optimized publication".  This can not meet WCAG-AA due to this specialized format but it is accessible to certain disability types.  This is why this is a URL, it can either point into our own specification referencing WCAG-A or AA, or it can point to any other specification which is an optimized format. 
Then based on the URL present we would interpret that URL and display 

Accessibility Conformance: EPUB Accessibility, WCAG A
or
Accessibility Conformance: EPUB Accessibility, WCAG AA
or
Accessibility Conformance: Optimized Navigable Audio Only
etc.

**Comment by gregoriopellegrino** [comment]
Good for the addition, I would not add the reference to the URL as it is specific to Schema.org (we can include it in the techniques).

I propose the text as follows:

**Value**: list of links, based on metadata

**Definition**
This metadata reports the conformity of the publication to one or more accessibility standards. The different standards can be listed comma separated.
In the case of EPUBs, if they conforms to EPUB Accessibility Guidelines, it means that they also conforms to WCAG (on which they are based). In this case the double value must be reported, for example: A_ccessibility Conformance: EPUB Accessibility, WCAG AA_

Incase the publication is for example an audiobook ie "optimized publication" this can not meet WCAG-AA due to this specialized format but it is accessible to certain disability types. In this case you can refer to particular specifications, for example: _Accessibility Conformance: Optimized Navigable Audio Only_

**From the note remove**: In this case, the conformsTo term must point to a URL that identifies the specification used to create the optimized publication.

**Comment by avneeshsingh** [comment]
@gregoriopellegrino for accuracy sake, the existing text points to double core conformsto metadata and not schema.org. 
I think we can use URLs of EPUB accessibility and optimized publications just as examples. But these should not look as a part of definition.

**Comment by gregoriopellegrino** [comment]
Ok actually, I meant that the fact that we represent that information in the form of URLs is related only to the metadata for EPUB, while ONIX uses codes from the 196 codelist:

- 02 for level A
- 03 for level AA

**Comment by avneeshsingh** [comment]
This is the pending issue that we should resolve before release of user experience guide.
If publication meets EPUB accessibility (WCAG a) then Right now the UI says
Accessibility conformance: WCAG AA
 
But what if the publication meets only WCAG a, and not EPUB Accessibility (WCAG A)? As per the current situation both will look the same in the user interface, the only difference will be the hyperlink.

Earlier we discussed to choose one of the following options for conformance to EPUB accessibility. We should make a decision to move this forward.
Accessibility Conformance: EPUB Accessibility, WCAG A
or
Accessibility Conformance: EPUB Accessibility, WCAG AA

**Comment by gregoriopellegrino** [comment]
I agree with:

- Accessibility Conformance: EPUB Accessibility, WCAG A
- Accessibility Conformance: EPUB Accessibility, WCAG AA

**Comment by GeorgeKerscher** [comment]
I agree that having two options, single A or Double AA is most appropriate. Some countries may only require single A.

**Comment by GeorgeKerscher** [comment]
Just a thought about adding the word plus, wich would make it makes it more clear that both EPUB accessibility and WCAG at single a or double A is what this conforms to as in:
"Accessibility Conformance: EPUB Accessibility, plus  WCAG AA"

**Comment by clapierre** [comment]
> If publication meets EPUB accessibility (WCAG a) then Right now the UI says
Accessibility conformance: WCAG AA

@avneeshsingh I assume you meant to say that the UI says Accessibility conformance: WCAG A 
not WCAG AA right?

I am also fine with having the two options the addition of adding "plus" I can go either way.

Question, Would they ever be just WCAG - A or WCAG - AA without being EPUB Accessible, and likewise could you claim EPUB Accessibility without also including either WCAG-A or AA?

**Comment by laudrain** [comment]
I agree with having both options.

About the "plus" word, I'm wondering if it is necessary as the text says : _The different standards can be listed comma separated_. 
Also a comma is easier to process in case of text analysis, and it is more international...

**Comment by avneeshsingh** [comment]
It looks we are more comfortable with comma instead of plus sign.
To summarize, if the publication conforms to EPUB accessibility, then UI should show:
- Accessibility Conformance: EPUB Accessibility, WCAG A
- Accessibility Conformance: EPUB Accessibility, WCAG AA
- Accessibility Conformance: EPUB Accessibility, WCAG AAA
I have added AAA also. Although ONIX codelist 196 dos not have code for EPUB Accessibility, WCAG AAA, the EPUB accessibility specifications allows it, so I think that tehre is no harm in keeping it in user experience guide.

And if the publication conforms to only WCAG and not EPUB Accessibility. Then UI can be:
Accessibility Conformance: WCAG A
Accessibility Conformance: WCAG AA
Accessibility Conformance: WCAG AAA
It is possible that some publishers only focus on WCAG conformance, so we should have provision for it also.

If this looks fine then we can edit the user experience guide and techniques accordingly.

**Comment by gregoriopellegrino** [comment]
For me it's fine. There is no way right now to express it in ONIX, where we have only "EPUB Accessibility Specification 1.0 A" or "EPUB Accessibility Specification 1.0 AA", but I can talk to EditEUR.

**Comment by avneeshsingh** [comment]
@clapierre and @gregoriopellegrino , this issue is open for some months and it looks that we are fine with this change. So, I would suggest to make this important change and close this issue.

**Comment by clapierre** [comment]
@avneeshsingh 
>And if the publication conforms to only WCAG and not EPUB Accessibility. Then UI can be:

However the 1.1 specification says these are all "MUST"'s

3.5.2 Publication Conformance
To indicate that an EPUB Publication conforms to the accessibility requirements of this specification, it MUST include a conformsTo property whose value MUST exactly match (i.e., both in case and spacing) the following pattern:

EPUB-A11Y-A11Y-VER_WCAG-WCAG-VER-WCAG-LVL

where:

**A11Y-VER**
EPUB Creators **MUST** specify the version number of the EPUB Accessibility specification the publication conforms to, not including the decimal points. EPUB Creators MUST use the value 11 to indicate this version of this specification.

**WCAG-VER**
EPUB Creators **MUST** specify the version number of WCAG the publication conforms to, not including the decimal points (e.g., 20 for WCAG 2.0 or 21 for WCAG 2.1).

**WCAG-LVL**
EPUB Creators **MUST** specify the WCAG conformance level the publication conforms to (e.g., A or AA).

The following conformance strings are valid as of publication of this specification:

EPUB-A11Y-11_WCAG-20-A
EPUB-A11Y-11_WCAG-20-AA
EPUB-A11Y-11_WCAG-20-AAA
EPUB-A11Y-11_WCAG-21-A
EPUB-A11Y-11_WCAG-21-AA
EPUB-A11Y-11_WCAG-21-AAA

So there is no way to have only a WCAG conformance without an EPUB Conformance unless we change the 1.1 spec to have the EPUB-A11Y-11 as optional right?

**Comment by mattgarrish** [comment]
> > And if the publication conforms to only WCAG and not EPUB Accessibility. Then UI can be:
> 
> However the 1.1 specification says these are all "MUST"'s

If you're only specifying conformance to WCAG you should be using the appropriate WCAG conformance URL, not trying to repurpose our accessibility URLs.

**Comment by clapierre** [comment]
Ok that makes sense thanks @mattgarrish 
But instead of the URLs pointing to the old IDPF's 1.0 spec #WCAGA etc.  which we had previously 
What URL would you use to specify WCAG 2.1 - AA?
Or WCAG 2.0 - A?

As far as I know there is no W3C URL's that can map to A/AA or AAA right?  Only the main specification

**Comment by avneeshsingh** [comment]
For WCAG conformance conformsto should be having a string. So, mapping to URL would not be needed.
Which means that we do not need to give example for mapping of URL for WCAG. We can give just three examples for EPUB accessibility, WCAG A to AAA

**Comment by clapierre** [comment]
ok so the string property would be "WCAG A" 
What about WCAG 2.0 vs. 2.1?

So would it be "WCAG 2.0 A"
"WCAG 2.1 AA"
for example?
Would we also include the old URL links to the IDPF 1.0 spec and link to something that conforms to another spec say for Talking Book.  Or is that just handled by the old 1.0 Techniques document and we don't need to cover them here in the 1.1 techniques

**Comment by avneeshsingh** [comment]
Right now User experience guide should be according to EPUB Accessibility 1.0, with old IDPF URL. We should update the guide soon after EPUB Accessibility 1.1 reaches the CR stage.

Regarding WCAG strings, I think we can stay away from specifying it too much here, because the guide informs how to interpret machine readable a11y metadata.
If there are URL specified by EPUB Accessibility 1.0 then the guide informs how these should be presented in UI.
But if there are strings or URL to another specification (not specified in EPUB Accessibility), then the string or URL  should be picked from metadata and displayed on the webpage.

**Comment by mattgarrish** [comment]
> As far as I know there is no W3C URL's that can map to A/AA or AAA right?

Not prior to 2.2, but do you need to get into content that doesn't fully conform? Why is that a problem for this guide to solve?

I read that you were asking if the accessibility spec forbids you from using other conformance strings/URLs. It doesn't, and I don't think we want to get into defining new conformance strings for WCAG. There's already a section in the standard on what you have to include, including the URL: https://www.w3.org/TR/WCAG21/#conformance-required

If I read @avneeshsingh's last comment right, I agree that for any other conformance statements, let the developer decide whether and how to display it. They can identify the conformance statements, so beyond that it's out of scope for us.
